const t=JSON.parse('{"key":"v-e4b4e492","path":"/blogs/front-end/js/return_in_tryCatch.html","title":"关于try/catch块中return的使用","lang":"en-US","frontmatter":{"title":"关于try/catch块中return的使用","date":"2023/05/07","categories":["JavaScript"],"tags":["JavaScript"]},"headers":[{"level":2,"title":"在try catch中使用return","slug":"在try-catch中使用return","link":"#在try-catch中使用return","children":[]},{"level":2,"title":"在嵌套try catch中使用return","slug":"在嵌套try-catch中使用return","link":"#在嵌套try-catch中使用return","children":[]}],"git":{"createdTime":1685675187000,"updatedTime":1685679948000,"contributors":[{"name":"Gavi","email":"oncwnuFugqfzgh-P2t1qKLPQ-GS0@git.weixin.qq.com","commits":2}]},"filePathRelative":"blogs/front-end/js/return_in_tryCatch.md"}');export{t as data};
